cmd/webhook/webhook.go

```diff
@@ -88,6 +89,7 @@ func init() {
 	Cmd.Flags().StringVar(&metricsAddr, "metrics-addr", defaultMetricsAddr, "metrics endpoint address")
 	Cmd.Flags().BoolVar(&offline, "offline", false, "do not connect to API server to retrieve imagePullSecrets")
 	Cmd.Flags().IntVar(&port, "port", defaultPort, "webhook server port")
+	Cmd.Flags().BoolVar(&ignoreError, "ignore-error", false, "only log the error; do not fail on the error")
```

Comment:
Since this flag only applies when running digester as a webhook and not as a KRM function, we should reword the flag description to make this clear.

How about "do not fail on webhook admission errors, just log them"?

Reply:
Also, let's name the flag "ignore-errors" (plural), to be consistent with Make.

Reply:
Hi @halvards, thank you for your review! I just fixed it as you suggested can you check it again?